.github/workflows/docker.yml

```diff
@@ -458,11 +458,7 @@ jobs:
             { image: all-spark-notebook, variant: default },
           ]
     needs: [aarch64-images-tag-push, x86_64-images-tag-push]
-    if: |
```

Comment:
This is not needed in this repo: https://github.com/jupyter/docker-stacks/pull/2091#discussion_r1485652584

And it actually makes some harm:
https://github.com/jupyter/docker-stacks/actions/runs/13362035735

In this run, building aarch64 `r-notebook` failed.
In my opinion, we should have not even tried to merge tags.
Instead, we tried, but only failed because manifest file was not available (because we were uploading it after running tests).

Instead, we should keep this condition simple and in accordance with other places.